chemicaltoolbox/rdkit/conformers.xml

```diff
@@ -0,0 +1,86 @@
+        #end if
+        -n '$num'
+        -a '$attempts'
+        -r '$rmsd'
+        -c '$conditional_cluster.cluster_method_opts'
+        #if $conditional_cluster.cluster_method_opts != 'none':
+            -t '$conditional_cluster.threshold'
+        #end if
+        -e '$emin'
+        -o '$conf_output'
+        --outformat '${conf_output.ext}'
+    ]]></command>
+    <inputs>
+        <conditional name="input">
+            <param name='input_opts' type='select' format='text' label="SMILES input type" help="Enter SMILES as either text or file.">
+                <option value='-i'>Text</option>
+                <option value='-f'>File</option>
+            </param>
+            <when value='-i'>
+                <param name="smiles" type="text" label="SMILES input" help="Enter SMILES for a compound.">
+                    <validator type='length' min='5'/>
+                </param>
+            </when>
+            <when value='-f'>
+                <param name="smiles" type="data" format="smi" label="Input file" help="File containing a single compound in SMILES format. Note only the first line of the file will be read, if the file contains multiple compounds."/>
+            </when>
+        </conditional>
+        <param name="num" type="integer" label="Number of conformers to generate" value="1"/>
+        <param name="rmsd" type="float" label="RMSD threshold" value="1.0" help="If a conformer is too similar to those already generated (below the RMSD threshold), it is discarded and regenerated."/>
+        <param name="attempts" type="integer" label="Number of attempts" value="1" help="Conformers which are below the RMSD threshold are regenerated. This parameter controls how many times this regeneration should be attempted before giving up. "/>
+        <conditional name="conditional_cluster">
+            <param name="cluster_method_opts" type="select" label="Cluster method" help="RMSD, TFD (torsion fingerprint deviation) or no clustering.">
+                <option value="none" selected="true">None</option>
+                <option value="rmsd">RMSD</option>
+                <option value="tfd">TFD</option>
+            </param>
+            <when value="none">
+            </when>
+            <when value="rmsd">
+                <param name="threshold" type="float" label="Cluster threshold" value="2.0" help="Default of 2.0 for RMSD"/>
+            </when>
+            <when value="tfd">
+                <param name="threshold" type="float" label="Cluster threshold" value="0.3" help="Default of 0.3 for TFD"/>
+            </when>
+        </conditional>
+        <param name="emin" type="integer" label="Energy minimisation iterations" value="0" help="Number of EM iterations using the MMFF force field. Default is 0 (i.e. no EM)."/>
+    </inputs>
+    <outputs>
+        <data name="conf_output" format="sdf" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="smiles_input" value="C(C1C(C(C(C(O1)O)O)O)O)O"/>
```

Comment:
I think the name here is wrong, it should be `smiles`. But not sure if this is what is causing tests to fail